zebra/zebra_vty.c

```diff
@@ -886,6 +1002,7 @@ DEFPY(ip_route,
 	  |(1-255)$distance                            \
 	  |vrf NAME                                    \
 	  |label WORD                                  \
+	  |table (1-4294967295)                        \
```

Comment:
Again, this needs to be a separate defun.

Reply:
I had done it for vrf node commands, but was thinking that it was ok for default vrf. so I updated it.